stables.js

```diff
@@ -12,4 +12,6 @@ let y = 1000;
 let z = 10;
 let a = 1.79769313486231 //What is the max number??
 const STABLE_MONTHLY_FEE = x*x*x*x*x*x*x*x*x*x*x*x*x*x*x*x*x*x*x*x*x*x*x*x*x*y*y*z*z*a;
-console.log(STABLE_MONTHLY_FEE)
\ No newline at end of file
+console.log("The stable will cost $" + (STABLE_MONTHLY_FEE*0.9) + "a month. This is 10% off")
+let horseNickname = "Tree Man"
+console.log(horseNickname)
```

Comment:
- great use of variables and strings together. 
- perhaps add a  string introducing your horses name/ nickname